plugin/pom.xml

```diff
@@ -45,7 +49,7 @@
     <dependency>
       <groupId>org.jvnet.hudson.plugins</groupId>
       <artifactId>analysis-core</artifactId>
-      <version>1.73-SNAPSHOT</version>
+      <version>1.73-beta-SNAPSHOT</version><!-- TODO: change to 1.73 before release -->
```

Comment:
All changes made on [analysis core PR](https://github.com/jenkinsci/analysis-core-plugin/pull/31) (upstream of this one) have been deployed as `1.73-beta-SNAPSHOT`, so it will not conflict with the real 1.73-SNAPSHOT (which should be linked to master branch, not from a PR).

In this way all downstream PRs compile successfully (in local environments and CI service) without requiring to checkout and compile locally the upstream PR in analysis-core. Once the upstream PR is merged in master, we can deploy the real 1.73-SNAPSHOT, then update all downstream PRs and merge then securely.